### PR TITLE
btrisolve: Make a Tensor method and update argument order

### PR DESCRIPTION
**Pull request by bamos**
Also update docs for btrifact and btrisolve to the newest interface.

### CODE REVIEW

File: torch/_torch_docs.py [hunk 1]
```diff
@@ -4395,7 +4396,8 @@
 
 Arguments:
     b (Tensor): RHS tensor.
-    A_LU (Tensor, IntTensor, fmt): LU-factorization of A from btrifact.
+    A_LU_data (Tensor): Pivoted LU factorization of A from btrifact.
```
Comment (apaszke) [inline torch/_torch_docs.py]: can you remove the `A_` prefix? They can be used as keywords so you should stick to the names from cwrap.

Comment (bamos) [inline torch/_torch_docs.py]: Sure, I amended my old commit and just force-pushed.

**Review by apaszke — APPROVED** [review]
Needs a subtree update first

**Comment by soumith** [comment]
merging this via https://github.com/pytorch/pytorch/pull/1117

**Pull request closed without merging**